client/components/themes-list/index.jsx

```diff
@@ -130,70 +139,148 @@ function ThemeBlock( props ) {
 	);
 }
 
-function Empty( props ) {
-	const { wpOrgThemes, emptyContent, searchTerm, upsellCardDisplayed, translate } = props;
+function Footer( props ) {
+	const { upsellCardDisplayed, translate } = props;
 	const selectedSite = useSelector( getSelectedSite );
-	const shouldUpgradeToInstallThemes = useSelector(
-		( state ) => ! siteHasFeature( state, selectedSite?.ID, FEATURE_INSTALL_THEMES )
-	);
-
-	const matchingThemes = useMemo(
-		() =>
-			wpOrgThemes?.filter(
-				( wpOrgTheme ) =>
-					wpOrgTheme?.name?.toLowerCase() === searchTerm.toLowerCase() ||
-					wpOrgTheme?.id?.toLowerCase() === searchTerm.toLowerCase()
-			) || [],
-		[ wpOrgThemes, searchTerm ]
+	const canInstallTheme = useSelector( ( state ) =>
+		siteHasFeature( state, selectedSite?.ID, FEATURE_INSTALL_THEMES )
 	);
+	const canGoToPatternAssembler = useViewportMatch( 'large' );
 
+	// Prevents the Upwork upsell from being visible, so it doesn't conflict with the actions below.
 	useEffect( () => {
-		if ( shouldUpgradeToInstallThemes && ! emptyContent ) {
+		if ( selectedSite ) {
 			upsellCardDisplayed( true );
 		} else {
 			upsellCardDisplayed( false );
 		}
 		return () => {
 			upsellCardDisplayed( false );
 		};
-	}, [ emptyContent, shouldUpgradeToInstallThemes, upsellCardDisplayed ] );
+	}, [ selectedSite, upsellCardDisplayed ] );
 
-	if ( emptyContent ) {
-		return emptyContent;
+	if ( ! selectedSite ) {
+		return null;
 	}
 
-	return shouldUpgradeToInstallThemes ? (
-		<div className="themes-list__empty-container">
-			{ matchingThemes.length ? (
-				<WPOrgMatchingThemes
-					matchingThemes={ matchingThemes }
-					selectedSite={ selectedSite }
-					{ ...props }
-				/>
-			) : (
-				<>
-					<div className="themes-list__not-found-text">
-						{ translate( 'No themes match your search' ) }
+	return (
+		<div className="themes-list__footer">
+			<div className="themes-list__footer-heading">
+				{ translate( "Can't find what you're looking for?" ) }
+			</div>
+			<div className="themes-list__footer-subheading">
+				{ translate( 'Here are a few more options' ) }
```

Comment:
Non-blocking nitpick. Should this have a colon? 🤔

Reply:
Yup, I wondered the same when I saw the proposed copy in https://github.com/Automattic/dotcom-forge/issues/1453. I think a colon it's more consistent with the rest of our writing style, so I'll add it.

Reply:
That's not a colon though! 😛

Reply:
Oh gosh, I was completely lost in translation and genuinely thought for a moment that a colon was the same than a dot 🤦